aicsimageio/tests/writers/test_ome_tiff_writer.py

```diff
@@ -4,17 +4,18 @@
 import unittest
 
 import numpy as np
+
 from aicsimageio.readers.ome_tiff_reader import OmeTiffReader
-from aicsimageio.writers import OmeTifWriter
+from aicsimageio.writers import OmeTiffWriter
```

Comment:
the fact that you had to rename these makes me think we had a bad merge some time before.  I could have sworn we changed them :)

Reply:
You are correct, we changed the readers but not the writers.